src/app/components/forms/forgot-password-form/forgot-password-form.component.scss

```diff
@@ -0,0 +1,20 @@
+@import '../../../styles/colors';
+
+.forgot-password-form-button-container {
+  text-align: center;
+
+  .error-message {
+    color: $color-cerise-500;
+  }
+
+  .forgot-password-form-button {
+    display: block;
+    margin: 10px auto;
+    min-width: 200px;
+  }
+
+  .forgot-password-form-link {
+    color: $color-accent-main;
```

Comment:
Properties should be ordered font-weight, color

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)